libvast/src/system/application.cpp

```diff
@@ -273,6 +285,11 @@ auto make_spawn_command() {
     opts()
       .add<size_t>("segments,s", "number of cached segments")
       .add<size_t>("max-segment-size,m", "maximum segment size in MB"));
+  spawn->add_subcommand(
+    "explorer", "creates a new explorer", "",
+    opts()
+      .add<vast::duration>("before,B", "timebox before each result")
+      .add<vast::duration>("after,A", "timebox after each result"));
```

Comment:
```suggestion
      .add<vast::duration>("after,A", "timebox after each result")
      .add<vast::duration>("before,B", "timebox before each result"));
```

Reply:
Also, do these even need the short options? `vast spawn` is hidden after all.

Reply:
I guess we don't need them, but the other spawn subcommands include them as well.

Reply:
Just leave them in. We have a story for removing them and can then do them all in one pass.